CGAL_Core/include/CGAL/CORE/CoreAux_impl.h

```diff
@@ -198,10 +177,8 @@ void core_error(std::string msg, std::string file, int lineno, bool err) {
   outFile.close();
   if (err) {
      char buf[65];
```

Comment:
`buf` is now useless. It should be removed to avoid a warning about an unused variable.